pkg/health/health.go

```diff
@@ -1,6 +1,7 @@
 package health
 
 import (
+	"log"
```

Comment:
Please use our vlogger package for consistency. (import log "github.com/F5Networks/k8s-bigip-ctlr/pkg/vlogger")

Reply:
Nevermind, I see this was changed in a later commit.